apps/gallery/serializers.py

```diff
@@ -21,6 +21,12 @@
 FILE_READONLY_FIELDS = ('metadata', 'mime_type',)
 
 
+class FileUploadSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = File
+        fields = ('__all__')
```

Comment:
Can you make sure existing logic are also here like FileSerializer? Maybe we can just create a new class using that instead?